packages/manager/modules/core/src/index.js

```diff
@@ -246,3 +246,21 @@ angular
   );
 
 export default moduleName;
+
+export const bootstrapApplication = () => {
+  return fetch('/engine/2api/configuration')
+    .then((response) => {
+      if (response.status === 401) {
+        window.location.assign(
+          `/auth?action=disconnect&onsuccess=${encodeURIComponent(
+            window.location.href,
+          )}`,
+        );
+      }
+      return response.json();
+    })
```

Comment:
I will add a catch to try to detect region using hostname to avoid breaking all applications if the API fails.

Reply:
Added 3 lines below.